controller/ctrlUsers.js

```diff
@@ -180,11 +184,52 @@ const registerUser = async (req, res, next) => {
   }
 };
 
+const updateAvatar = async (req, res, next) => {
+  const { _id } = req.user;
+  const avatarURL = `/avatars/av_${_id}.png`;
```

Comment:
`./avtars/...` (czyli z `.` na poczatku) bedzie lepsze, bo w tej chwili mowisz mu, zeby szukal `/avatars` w root dysku, na ktorym sa pliki (np. `C:/avatars`). To na serwerze moze _backfire_.

Reply:
Naprawione